pages/index.js

```diff
@@ -3,18 +3,19 @@ import Sponsors from '/components/website/home/Sponsors';
 import Hackathon from '/components/website/home/Hackathon';
 import Speakers from '/components/website/home/Speakers';
 
+import Navbar from '/components/website/utils/Navbar';
 import Schedule from '/components/website/utils/Schedule';
 import Footer from '/components/website/utils/Footer';
 
 export default function Index() {
   return (
-    <>
-      <Hero />
-      <Schedule color="tertiary" detailed={false}/>
-      <Sponsors/>
-      <Hackathon />
-      <Speakers />
-      <Footer color="secondary" />
-    </>
+      <Navbar bg_color="secondary" button="quinary" fg_color="black">
```

Comment:
In JavaScript, variables (in this case props) should be camelCase.
```suggestion
      <Navbar bgColor="secondary" button="quinary" fgColor="black">
```